packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Providers/FixedEnumSerializationProvider.cs

```diff
@@ -7,32 +7,36 @@
 using System.IO;
 using System.Linq;
 using Microsoft.Generator.CSharp.Expressions;
+using Microsoft.Generator.CSharp.Input;
 using Microsoft.Generator.CSharp.Primitives;
+using Microsoft.Generator.CSharp.Providers;
 using Microsoft.Generator.CSharp.Snippets;
 using Microsoft.Generator.CSharp.Statements;
 using static Microsoft.Generator.CSharp.Snippets.Snippet;
 
-namespace Microsoft.Generator.CSharp.Providers
+namespace Microsoft.Generator.CSharp.ClientModel.Providers
```

Comment:
so we decide that the "serialization for fixed and extensible enums" also belong to Scm library, right?

Reply:
yes serialization belongs to scm library